CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## XX.XX.XX - 20XX-XX-XX
 
-*[ADDED][9158](https://github.com/stripe/stripe-android/pull/9158) Add `requireCvcRecollection` param to `IntentConfiguration`
+*[ADDED][9158](https://github.com/stripe/stripe-android/pull/9158) Add `requireCvcRecollection` param to `IntentConfiguration`. Currently in private beta. This is a breaking change for merchants using `PaymentSheet.Builder.cvcRecollectionEnabledCallback` and `PaymentSheet.FlowController.Builder.cvcRecollectionEnabledCallback`
```

Comment:
I think we could reword this to be a little more clear.

A few things:
* Let's update the tag from [ADDED] -> [BREAKING]
* Let's start with something like "Updates to CVC recollection APIs, currently in private beta", then list the changes
* Let's also mention (probably at the end) this is still in private beta, and requires the experimental API annotation.

Reply:
Updated 👍